examples/purchaseTesterTypescript/app/screens/HomeScreen.tsx

```diff
@@ -190,6 +179,12 @@ const HomeScreen: React.FC<Props> = ({
                   Show In-App messages
                 </Text>
             </TouchableOpacity>
+            <TouchableOpacity
+              onPress={() => navigation.navigate('Paywall')}>
+              <Text style={styles.otherActions}>
+                Go to Paywall Screen
+              </Text>
+            </TouchableOpacity>
```

Comment:
Added this button at the end of the screen to open a new screen that has the RNPaywall component